recipes/hwm/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "hwm" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hwm-{{ version }}.tar.gz
+  sha256: 5f09159edf703390f4d3110d4737b2f2c47754a8bf11c2b0993e67270773f8c8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - numpy <2
+    - scipy >=1.7.0
+    - scikit-learn >=1.2.0
+    - pandas >=1.3.0
+    - joblib >=1.2.0
+    - matplotlib-base >=3.5.2  
+    
+test:
+  imports:
+    - hwm
+    - hwm.api
+    - hwm.compat
+    - hwm.datasets
+    - hwm.estimators
+    - hwm.utils
+  commands:
+    - python -c "import hwm; print(hwm.__version__)"
```

What you have is a duplicate of the import test above. We need `pip check` here.

```suggestion
    - pip check
```